bin/syclcc-clang

```diff
@@ -1193,7 +1236,16 @@ class compiler:
 
     for backend in self._targets:
       if backend == "omp":
-        self._backends.append(omp_invocation(config))
+        if config.use_accelerated_cpu or \
+                (not "cuda-nvcxx" in config.targets
+                 and not sys.platform.startswith("win32") and not sys.platform.startswith("darwin")):
```

Comment:
Should we not also check whether hipsycl-cpu-cxx is clang here?

Reply:
fixed.